From 69a4a718044d9e1e8860618633857efc34eda6ff Mon Sep 17 00:00:00 2001
From: Owen D <student@u.boisestate.edu>
Date: Fri, 9 May 2025 14:05:00 -0600
Subject: [PATCH] Added my name

---
 README.md | 1 +
 1 file changed, 1 insertion(+)

diff --git a/README.md b/README.md
index 181ad22..73f3a2f 100644
--- a/README.md
+++ b/README.md
@@ -17,3 +17,4 @@
 * Sahar Sepehr
 * Tanner Silva
 * Ryan Posey
+* Owen DeChambeau
-- 
2.25.1

